website/docs/r/budgets_budget.html.markdown

```diff
@@ -58,6 +58,46 @@ resource "aws_budgets_budget" "s3" {
 }
 ```
 
+Create a Savings Plan Utilization Budget
+```hcl
+resource "aws_budgets_budget" "savings_plan_utilization" {
```

Comment:
I tried running the below example configuration and it threw an error:

```hcl
resource "aws_budgets_budget" "savings_plan_utilization" {
  name = "bflad-savingsplan"
  time_period_start = "2020-01-01_12:00"
  time_unit = "MONTHLY"

  budget_type  = "SAVINGS_PLANS_UTILIZATION"
  limit_amount = "100.0"
  limit_unit   = "PERCENTAGE"
}
```

```
Error: create budget failed: InvalidParameterException: [Exception=InvalidParameterException] Failed to call CreateBudget for [AccountId: --OMITTED--] - Unable to create/update budget - SAVINGS_PLANS_UTILIZATION budget type only supports the following costTypes combination: { includeSubscription: true, includeTax: false, useBlended: false, includeRefund: false, includeCredit: false, includeUpfront: false, includeRecurring: false, includeOtherSubscription: false, includeSupport: false, includeDiscount: false, useAmortized: false }
```

Adding the `cost_types` information, similar to RI_UTILIZATION, fixed it.

```hcl
resource "aws_budgets_budget" "savings_plan_utilization" {
  name = "bflad-savingsplan"
  time_period_start = "2020-01-01_12:00"
  time_unit = "MONTHLY"

  budget_type  = "SAVINGS_PLANS_UTILIZATION"
  limit_amount = "100.0"
  limit_unit   = "PERCENTAGE"

  cost_types {
    include_credit             = false
    include_discount           = false
    include_other_subscription = false
    include_recurring          = false
    include_refund             = false
    include_subscription       = true
    include_support            = false
    include_tax                = false
    include_upfront            = false
    use_blended                = false
  }
}
```